src/bigint.cpp

```diff
@@ -169,6 +169,16 @@ Bigint &Bigint::operator-=(Bigint const &b)
     }
     if (dif < 0) positive = false;
 
+	if (number.size() > 1)
```

Comment:
How is this better than a regular while loop? The same condition is called either way.